source/stylesheets/refills/_device.scss

```diff
@@ -1,35 +1,19 @@
 .device-background {
-  ///////////////////////////////////////////////////////////////////////////////////
-  $base-border-radius: 3px !default;
-  $base-accent-color: #477DCA !default;
-  $large-screen: em(860) !default;
-
-  p {
-    line-height: 1.5em;
-  }
-
-  h4 {
-    font-size: 1.5em;
-    margin: 0;
-  }
-  //////////////////////////////////////////////////////////////////////////////////
-
   $device-padding-vertical: 5em;
   $device-padding-horizontal: 0.4em;
   $device-screen-width: 14em;
   $device-screen-height: 25em;
   $device-background: darken(gray, 40%);
   $device-backside-background: darken($device-background, 10%);
-  $device-text-color: white;
-  $device-background-top: #162C4C; 
+  $device-text-color: $base-font-color-on-box;
+  $device-background-top: #162C4C;
   $device-background-bottom: #0A120D;
   $gradient-angle: 10deg;
-  $device-image: 'https://raw.githubusercontent.com/thoughtbot/refills/master/source/images/mountains.png';
   $device-screen-image: 'https://raw.githubusercontent.com/thoughtbot/refills/master/source/images/cosmin_capitanu_screen.jpg';
 
-  @include background(url($device-image), linear-gradient($gradient-angle, $device-background-bottom, $device-background-top), no-repeat $device-background-top scroll);
-  background-repeat: no-repeat;
-  background-size: cover;
+  background: $base-box-background;
+  border-radius: $base-border-radius;
```

Comment:
Properties should be ordered background, border, border-radius, min-height, width